src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -593,6 +596,29 @@ public void clearCachedCalenderService() {
     this.calendarService = null;
   }
 
+  public OpeningHoursCalendarService getOpeningHoursCalendarService() {
+    if (openingHoursCalendarService == null) {
+      if (transitServiceStarts == Long.MAX_VALUE) {
+        // transit start/end has not been set, default to -1 year and +3 years
+        this.openingHoursCalendarService =
+          new OpeningHoursCalendarService(
+            deduplicator,
+            LocalDate.now().minusYears(1),
+            LocalDate.now().plusYears(3)
+          );
+      } else {
```

Comment:
I don't have a strong opinion on this. I think they already have the same default values set but some tests didn't use those default values which caused the tests to fail.

Reply:
I had to change this in https://github.com/opentripplanner/OpenTripPlanner/pull/4228/commits/aeb0cc7490bbca2b11276893bbb78872e63322cb because of the transit model refactoring. I'm not sure if the place were I set it from is the best nor if it should be done through a setter method or directly from the Graph constructor. While doing the changes I realised that the earlier version was probably not working as intended for the OSM part anyway because the transit start/end was set afterwards while reading in the transit data so for the OSM opening hours it was probably always using the defaults.